### PR TITLE
Add default labels to issue templates

### LINKED ISSUE #1631: Update issue templates to add default labels automatically
**Is your feature request related to a problem? Please describe.**
The current issue templates have a checkbox to remind authors to add the default labels to the issue. As this is a manual step it is easy to forget, especially when raising multiple issues.

**Describe the solution you'd like**
Update the issue templates to add the following labels by default

- Feature request -> refinement needed
- Bug report -> bug

**Describe alternatives you've considered**
Not doing this - but why would you not ¯\_(ツ)_/¯

**Testing notes**
No testing required - other than making sure labels are added automatically to issues ;)

**Additional context**
https://github.community/t5/How-to-use-Git-and-GitHub/Apply-labels-automatically-at-issue-creation/m-p/15909/highlight/true#M4903

- [x] Initially labelled with ["Refinement needed"](https://github.com/bbc/simorgh/labels/Refinement%20Needed)
^^ Hopefully the last time this is needed ;)

### PR DESCRIPTION
**Pull request by rossgaskell**
Resolves #1631 

**Overall change:** Add default labels to issue templates and remove checkbox

**Code changes:**

- Add `bug` label to bug report template
- Add `Refinement Needed` label to bug report template

---

- [x] I have assigned myself to this PR and the corresponding issues
- [N/A] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Review by chris-hinds — APPROVED** [review]
Ace! 👍

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 name: Feature request
 about: Suggest an idea for this project
-
+labels: Refinement Needed
```
Comment (dr3) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]: Are we sure this doesn't need quotes, as its 2 words?

Comment (rossgaskell) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: Apparently it's only required if the label has special characters or emojis.

Comment (dr3) [reply .github/ISSUE_TEMPLATE/feature_request.md:4]: Ah Yep, [it uses commas](https://github.community/t5/How-to-use-Git-and-GitHub/Apply-labels-automatically-at-issue-creation/m-p/15909/highlight/true#M4903)

**Review by dr3 — APPROVED** [review]

**Pull request merged**